src/test/resources/META-INF/microprofile-config.properties

```diff
@@ -16,3 +16,4 @@ test.filesDir=/tmp/dataverse
 dataverse.files.directory=${test.filesDir}
 dataverse.files.uploads=${test.filesDir}/uploads
 dataverse.files.docroot=${test.filesDir}/docroot
+payara.security.openid.sessionScopedConfiguration=true
```

Comment:
Please explain why this is here in a comment within the file. Also, this should be referenced as a necessity for multi-tenant provider things with a link to the Payara guides and from the class where the `@OpenIdAuthenticationDefinition` annotation resides (existance + reasoning).